File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    var answer = [];
    let [minX, minY, maxX, maxY] = [51, 51, 0, 0];
    for (let x = 0; x<wallpaper.length; x++) {
        for (let y = 0; y<wallpaper[x].length; y++) {
            if (wallpaper[x][y]==='#') {
                minX = Math.min(minX, x);
                minY = Math.min(minY, y);
                maxX = Math.max(maxX, x);
                maxY = Math.max(maxY, y);
            }
        }
    }
    answer = [minX, minY, maxX+1, maxY+1];
    return answer;
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = {};
        int lux = 51, luy = 51, rdx = 0, rdy = 0;
        for (int i=0; i<wallpaper.length; i++) {
            for (int j=0; j<wallpaper[i].length(); j++) {
                if (wallpaper[i].charAt(j)=='#') {
                    if (i < lux) {
                        lux = i;
                    }
                    if (i > rdx) {
                        rdx = i;
                    }
                    if (j < luy) {
                        luy = j;
                    }
                    if (j > rdy) {
                        rdy = j;
                    }
                }
            }
        }
        answer = new int[] {lux, luy, rdx+1, rdy+1};
        return answer;
    }
}